packages/flex-plugin-scripts/src/scripts/build.ts

```diff
@@ -85,6 +86,7 @@ const build = async (...argv: string[]): Promise<void> => {
 
   try {
     const { warnings, bundles } = await _runWebpack();
+    logger.info(warnings);
```

Comment:
Do we need this log or was it for testing?

Reply:
removed